src/cmake/GeosxOptions.cmake

```diff
@@ -123,6 +123,15 @@ blt_append_custom_compiler_flag( FLAGS_VAR GEOSX_NINJA_FLAGS
                                  CLANG   "-fcolor-diagnostics"
                                )
 
+# clang-13 and gcc complains about unused-but-set variable.
+include(CheckCXXCompilerFlag)
+CHECK_CXX_COMPILER_FLAG("-Wunused-but-set-variable" CXX_UNUSED_BUT_SET_VAR)
+if (ENABLE_GBENCHMARK)
+    blt_add_target_compile_flags(TO benchmark
+                                FLAGS $<$<AND:${CXX_UNUSED_BUT_SET_VAR},$<COMPILE_LANGUAGE:CXX>>:-Wno-unused-but-set-variable>
+                                )
+endif()
```

Comment:
Can you remind me what was the problem that lead to this patch

Reply:
with clang 13.4 (image xcode13.4) this flag detects an unused variable in gbenchmark (you can find the error in the logs). It will also happen with a recent gcc (mine is gcc 12.2). I just disable it for the  benchmark target.